packages/cli/test/scripts/transfer.test.js

```diff
@@ -58,8 +62,8 @@ contract('transfer script', function(accounts) {
           unit: 'ether',
         });
 
-        (await ZWeb3.getBalance(sender)).should.not.eq((100e18).toString());
-        (await ZWeb3.getBalance(receiver)).should.eq((110e18).toString());
+        (await ZWeb3.getBalance(sender)).should.not.eq(new BN(senderBalance).minus(10e18).toString());
```

Comment:
Shouldn't this be `should.eq` (without the `not`)? The old test was checking simply that the balance had changed, but it looks like you're trying to assert that it was reduced by the correct amount/

It won't match exactly however because the transaction cost gas. A quick hack for this is to make the transaction have a gas price of 0, but that may not work on all clients.